packages/mobile/src/components/core/CardList.tsx

```diff
@@ -58,6 +69,9 @@ export function CardList<ItemT extends {}>(props: CardListProps<ItemT>) {
   } = props
 
   const styles = useStyles()
+  const { isEnabled: isSearchExploreEnabled } = useFeatureFlag(
```

Comment:
my guess is that we dont want all card-lists to be carousels right, there should be a new prop to card-list enabling carousel behaviour?

Reply:
I think its probably best to create a new component "CardListCarousel" that uses CardList under the hood and passes new props to it like root style + updated renderItem?

Reply:
Note, after reviewing your usage, i think it's probably best to actually have card-list have a "carousel" variant, so we dont have to create 4x components for UserListCarousel, TrackCarousel etc, they can just all use a "carousel" prop and "it just works"